android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -1678,37 +1680,6 @@ public void onMediaFilesUpdated(Url pageUrl, PlaylistItem[] playlistItems) {
         }
     }
 
-    @Override
-    protected void onDetachedFromWindow() {
-        super.onDetachedFromWindow();
-        // Remove the observer when the layout is detached from the window.
-        if (mFullscreenManager != null) {
-            mFullscreenManager.removeObserver(this);
-        }
-    }
-
-    // FullscreenManager.Observer method.
-    @Override
-    public void onEnterFullscreen(Tab tab, FullscreenOptions options) {
-        final WebContents webContents = tab.getWebContents();
-        if (webContents != null
-                && BraveYouTubeScriptInjectorNativeHelper.hasFullscreenBeenRequested(webContents)) {
-            MediaSession mediaSession = MediaSession.fromWebContents(webContents);
-            if (mediaSession != null) {
-                mediaSession.resume();
-            }
-            Activity activity = tab.getWindowAndroid().getActivity().get();
-            if (activity != null) {
-                try {
-                    activity.enterPictureInPictureMode(
-                            new PictureInPictureParams.Builder().build());
-                } catch (IllegalStateException | IllegalArgumentException e) {
-                    Log.e(TAG, "Error entering picture in picture mode.", e);
-                }
-            }
-        }
-    }
-
```

Comment:
All this code is not needed anymore as we now call `BraveYouTubeScriptInjectorNativeHelper#enterPictureInPicture()` from native.